web/source/dom/touchAliasElement.ts

```diff
@@ -77,7 +77,7 @@ namespace com.keyman.dom {
         console.info(`Target's grandparent: ${elementDescriber(target.parentElement.parentElement)}`);
       }
       // Triggers the actual Sentry log (on pages that include Sentry logging)
-      throw new Error(`Unexpected touch-start event target.`);
+      //throw new Error(`Unexpected touch-start event target.`);
```

Comment:
No `if(!keyman.isEmbedded)` check?  I'll admit I should have done that much originally; it'd still greatly help to get events from keymanweb.com in the meantime.

Reply:
There's a few events from keymanweb.com that can be analyzed

http://sentry.keyman.com/organizations/keyman/issues/5444/?project=10&query=is%3Aunresolved

Reply:
I think we should have some events from keymanweb.com. Given it's causing errors, let's not leave it running.